runtime/doc/filetype.txt

```diff
@@ -544,6 +544,8 @@ CTRL-T                    Jump back to the location that the manpage was
                           opened from.
 q                         :quit if invoked as $MANPAGER, otherwise :close.
 
+To see the table of contents, type <M-]>.
```

Comment:
For the man.vim case, this should be listed like the other mappings above.

    META-]                           Show the manpage outline in the |location-list|.